pages/osx/oathtool.md

```diff
@@ -1,15 +1,16 @@
 # oathtool
 
 > OATH one-time password tool.
+> More information: <https://www.mankier.com/1/oathtool>.
```

Comment:
This is the author's page
```suggestion
> More information: <https://www.nongnu.org/oath-toolkit/oathtool.1.html>.
```

This page should be moved to `common/` because is available in Linux too.